src/EnergyPlus/DisplacementVentMgr.cc

```diff
@@ -715,60 +715,72 @@ void CalcUCSDDV(EnergyPlusData &state, int const ZoneNum) // Which Zonenum
             } else {
 
                 if (state.dataRoomAirMod->SurfParametersCVDV(state.dataRoomAirMod->AirflowNetworkSurfaceUCSDCV(Loop, ZoneNum)).Zmax +
-                            Zone(state.dataSurface
+                            state.dataHeatBal
+                                ->Zone(
+                                    state.dataSurface
                                      ->Surface(
                                          state.afn->MultizoneSurfaceData(state.dataRoomAirMod->AirflowNetworkSurfaceUCSDCV(Loop, ZoneNum)).SurfNum)
                                      .Zone)
                                 .OriginZ -
-                            Zone(ZoneNum).OriginZ <
+                            zone.OriginZ <
                         0.8 &&
```

Comment:
For example here. The outer `Zone` needs to be expanded while the inner `Zone(ZoneNum)` needs to be reduced. It's easy to see now but the first time through caused diffs and had to revert to get back to working.